dash/App/Accounts/index.js

```diff
@@ -27,10 +27,10 @@ class AddAccounts extends React.Component {
       </div>
     )
   }
-  renderAddKeyring() {
+  renderAddKeyring({ accountData }) {
     return (
       <div className='addAccounts cardShow'>
-        <AddRing close={this.props.close} />
+        <AddRing close={this.props.close} accountData={accountData} />
```

Comment:
what is this new `accountData`? is this the navigation data for the entry component?